src/wp-admin/nav-menus.php

```diff
@@ -510,7 +531,13 @@
 			// Set menu locations.
 			set_theme_mod( 'nav_menu_locations', $menu_locations );
 
-			$messages[] = '<div id="message" class="updated notice is-dismissible"><p>' . __( 'Menu locations updated.' ) . '</p></div>';
+			$messages[] = wp_admin_notice(
+				__( 'Menu locations updated.' ),
+				array(
+					'additional_classes' => array( 'updated' ),
+					'dismissible'        => true,
+				)
+			);
```

Comment:
```suggestion
			$messages[] = wp_get_admin_notice(
				__( 'Menu locations updated.' ),
				array(
					'id'                 => 'message',
					'additional_classes' => array( 'updated' ),
					'dismissible'        => true,
				)
			);
```
1. Should use `wp_get_admin_notice()` here.
2. The div previously had an ID of `'message'`.